### ISSUE TITLE
Projects: Task Board

### ISSUE THREAD

**Issue by elliotBraem** [NEARBuilders/gateway#171]
Is an adaptation of [efiz.near/widget/Gigs.Template](https://near.social/efiz.near/widget/Gigs.Template) and [efiz.near/widget/Gigs.Board](https://near.social/mob.near/widget/WidgetSource?src=efiz.near/widget/Gigs.Board)

<img width="855" alt="Screenshot 2024-02-20 at 6 45 54 AM" src="https://github.com/NEARBuilders/gateway/assets/16282460/e9e8fea5-f932-4c66-8363-1497c945f1a1">

Keep theming defined in the parent app.

When a task is clicked, show the modal:

<img width="486" alt="Screenshot 2024-02-20 at 6 48 29 AM" src="https://github.com/NEARBuilders/gateway/assets/16282460/8f5224dd-cc46-44cf-869e-d02ceec02776">

When a "add task" is clicked, show the modal:

<img width="450" alt="Screenshot 2024-02-20 at 6 49 00 AM" src="https://github.com/NEARBuilders/gateway/assets/16282460/668a511e-5c48-441d-929b-eb93c6cb43ab">

### Acceptance Criteria
- [ ] Theming is defined in the root

**Comment by itexpert120** [comment]
Can you explain in detail how this works?

**Comment by elliotBraem** [comment]
@itexpert120 It's a [near-social-bridge](https://github.com/wpdas/near-social-bridge) component -- so it's a widget that holds an iframe, and the iframe holds a react app.

See the documentation at top of file [here](https://near.social/mob.near/widget/WidgetSource?src=efiz.near/widget/Gigs.Board)

The gigs board react app itself can be found here: https://github.com/near-everything/gigs-board


So in short, this story is mostly just:

- [ ] getting variables in the react app's CSS so the root widget can configure theme
- [ ] creating a "handler" for when a task is clicked, bring up modal
- [ ] configuring with task data (we can start with Social.index("hashtag", "task")
